src/tribler-core/tribler_core/components/tag/db/tag_db.py

```diff
@@ -7,18 +7,25 @@
 from pony.utils import between
 
 from tribler_core.components.tag.community.tag_payload import TagOperation, TagOperationEnum
+from tribler_core.utilities.pony_utils import get_or_create
 from tribler_core.utilities.unicode import hexlify
 
+CLOCK_START_VALUE = 0
+
+# we picked `-1` as a value because it is allows manually created tags get a higher priority
+CLOCK_FOR_AUTOGENERATED_TAGS = CLOCK_START_VALUE - 1
```

Comment:
Probably this logic is unnecessary and can be removed, as we also have `is_auto_generated` field

Reply:
Fixed.